docs/content/latest/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md

```diff
@@ -105,6 +105,34 @@ When configured, Yugabyte Platform users are able to login by specifying the com
 
 For additional information, see [Update a configuration key](https://yugabyte.stoplight.io/docs/yugabyte-platform/b3A6MTg5NDc2OTY-update-a-configuration-key).
 
+In addition to the preceding parameters, you may choose to specify parameters for the service account credentials. This  would be helpful in certain scenarios. For example, Windows Active Directory (AD) server does not typically provide regular users with query permissions for the LDAP server. Setting service account credentials would enable these users to query the LDAP server ([the `yugabytePlatformRole` attribute](#define-the-yugabyte-platform-role) would be read and set it accordingly).
```

Comment:
Just one minor addition: Can we also mention that the service account should have enough permissions to query the LDAP server, find users and read the user attributes?